### ISSUE TITLE
Support stdin or pipe env file

### ISSUE THREAD

**Issue by Abdillah** [tesujimath/nu_plugin_bash_env#1]
Enable env parsing via stdin instead of parameter path.

```
cat simple.env | bash-env

# a bash script
cat env.sh | bash-env
```

Some tooling (binary written in languages) sometime only output bash script to be`eval`-ed:

- `fnm env` (node manager)
- `opam env`
- `gnupg agent`

## Current State
Currently, invoking `bash-env` on end pipe displays an error.

```
$ (opam env) | bash-env
Error: nu::parser::missing_positional

  × Missing required positional argument.
   ╭─[entry #6:1:1]
 1 │ (opam env) | bash-env
   ╰────
  help: Usage: bash-env <path> . Use `--help` for more information.
```

**Comment by tesujimath** [comment]
That's a good idea. I'll look into it!

**Comment by Abdillah** [comment]
I need this soon, so I tried on and create a PR.

For my usage it is good enough, maybe on some cases it doesn't work. Hope you can try and review it.

**Comment by tesujimath** [comment]
Great, thanks. I'll have a look, but probably not immediately.

**Comment by tesujimath** [comment]
Fixed in 0.4.0.  Thanks @Abdillah
